Apps/Src/Voltage.c

```diff
@@ -99,7 +99,14 @@ SafetyStatus Voltage_CheckStatus(void){
  */
 SafetyStatus *Voltage_GetModulesInDanger(void){
 	static SafetyStatus checks[NUM_BATTERY_MODULES];
-	uint32_t open_wires = Voltage_GetOpenWire();
+	Voltage_GetOpenWire();
+	uint32_t open_wires = 0;
```

Comment:
Can you change this to a `static` global or a `static` global array, similar to `VoltageVal`? This way we can have a buffer to store the most recent open wire measurements